pkg/api/controller_test.go

```diff
@@ -1907,6 +1909,27 @@ func TestCrossRepoMount(t *testing.T) {
 		So(err, ShouldBeNil)
 		So(postResponse.StatusCode(), ShouldEqual, 201)
 
+		// We have uploaded a blob and since we have provided digest it should be full blob upload and there should be entry
+		// in cache, now try mount blob request status and it should be 201 because now blob is present in cache
+		// and it should do hard link.
+
+		params["mount"] = digest
```

Comment:
remove this "mount" param?

Reply:
I am setting params["mount"] = "sha:" some lines above, so before making next request i am assigning valid digest.